pkg/types/report.go

```diff
@@ -25,6 +26,8 @@ type Report struct {
 	ArtifactType  ftypes.ArtifactType `json:",omitempty"`
 	Metadata      Metadata            `json:",omitempty"`
 	Results       Results             `json:",omitempty"`
+	StartTime     time.Time           `json:"-"`
+	EndTime       time.Time           `json:"-"`
```

Comment:
I'm not sure, but perhaps we should also add `StartTime` and `EndTime` in `json` format.
@knqyf263 wdyt?

Reply:
I'm ok with that.